core/modules/ccontrol/testCControl.cc

```diff
@@ -686,6 +707,246 @@ static const std::vector<Antlr4CompareQueries> ANTLR4_COMPARE_QUERIES = {
         nullptr,
         "SELECT Source.sourceId,Source.objectId FROM Source WHERE Source.objectId IN(386942193651348) ORDER BY Source.sourceId"
     ),
+
+    // tests the null-safe equals operator
+    Antlr4CompareQueries(
+        "SELECT ra_PS FROM Object WHERE objectId<=>417857368235490",
+        "SELECT ra_PS FROM Object WHERE objectId = 417857368235490",
+        [](query::SelectStmt::Ptr const & selectStatement) {
+            // change the equals op to be the null safe equals op
+            auto whereClauseRef = selectStatement->getWhereClause();
```

Comment:
it doesn't really work to make `whereClauseRef` const, because it continues getting member objects, until finally an object is in fact changed, so the references/pointers getting there must be non-const.